spacy/tests/tok2vec.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from thinc.api import Model
+from thinc.types import Floats2d
+
+from spacy.tokens import Doc
+from spacy.util import registry
+
+
+@registry.architectures("LazyInitTok2Vec.v1")
```

Comment:
Maybe we should add a "test." prefix just to be fully clear this isn't meant to be used in any way?